tests/unit/system/classes/CombineAssetsTest.php

```diff
@@ -49,7 +50,8 @@ public function testCombine()
         $this->assertNotNull($url);
         $this->assertRegExp('/\w+[-]\d+/i', $url); // Must contain hash-number
 
-        $url = $combiner->combine([
+        $url = $combiner->combine(
+            [
             'assets/js/script1.js',
             'assets/js/script2.js'
```

Comment:
Indentation on these two lines needs to be added

Reply:
@LukeTowers done.